{
  try {
    return delegate.call();
  }
 catch (  Throwable t) {
    this.errorHandler.handleError(t);
    return null;
  }
}
